devolving_music/lib/elo_scoring.py

```diff
@@ -0,0 +1,41 @@
+# Python 3 program for Elo Rating
+import math
+
+# Function to calculate the Probability
+def Probability(rating1, rating2):
+
+	return 1.0 * 1.0 / (1 + 1.0 * math.pow(10, 1.0 * (rating1 - rating2) / 400))
```

Comment:
```suggestion
	return 1 / (1 + 10.0**(rating1 - rating2) / 400)
```

Why are there so many multiplications by `1.0` in this function? If you need to convert something to a float use `float()` so it's obvious what you're trying to do, but even so I don't think all of the `1.0`s are necessary.

Reply:
jk, I meant `1 / (1 + 10**((rating1 - rating2) / 400))`